apps/nextjs/tests-e2e/config/config.ts

```diff
@@ -5,7 +5,8 @@ dotenv.config({ path: path.join(process.cwd(), ".env") });
 
 export const TEST_USER_EMAIL = process.env.TEST_USER_EMAIL;
 export const TEST_USER_PASSWORD = process.env.TEST_USER_PASSWORD;
+export const PORT = process.env.PORT ?? 2525;
 export const TEST_BASE_URL =
-  process.env.TEST_BASE_URL || "http://localhost:2525";
+  process.env.TEST_BASE_URL || `http://localhost:${PORT}`;
```

Comment:
Allow us to specify the port where the server is running. Eg. run the server using a built version of the app rather than the dev server